rpc/src/types/unistring.rs

```diff
@@ -0,0 +1,30 @@
+use serde::Deserialize;
+
+#[derive(Deserialize)]
+#[serde(untagged)]
+pub enum Unistring {
+  ValidUtf8(String),
+  InvalidUtf8 { invalid_utf8_string: Vec<u8> },
```

Comment:
Nitpicky, but maybe for consistency this should just be `InvalidUtf8(Vec<u8>)`, but I'm kinda on the line since it might not be clear what its supposed to be. @astherath thoughts?